### DISCUSSION TITLE
Approach diferente al ejercicio

### DISCUSSION THREAD

**Discussion by nozerorma** [nozerorma/transcriptomic-final-exercise#4] [Show and tell]
Hola @SGMartin ! Como dijisteis que apreciaríais el hecho de que nos pegásemos con diferentes herramientas, he creado un pipeline automatizado para muestras de SRA tipo paired-end, en el que poder decidir qué conjunto de herramientas se prefiere elegir y que tipo de workflow se prefiere seguir en función de las muestras con las que se trabaja, incluyendo las que aparecen en el readme. Aunque es un approach diferente al que se pedía para la primera parte del trabajo, espero que os parezca interesante.

Para la segunda parte, mi intención es hacer algo del estilo con DESeq2 y EdgeR, aunque dependerá de lo mucho o poco que me pegue con ellos.

**Comment by SGMartin** [comment]
Si te sobra tiempo ;) :

* [snakemake](https://snakemake.readthedocs.io/en/stable/)
* [Nextflow](https://www.nextflow.io/)

**Reply by nozerorma** [reply]
Hola @SGMartin , al final no llegué a ponerme a intentar hacer un workflow en ninguna de las dos herramientas, aunque estuve indagando bastante en Nextflow en el módulo de micro.

El trabajo está terminado, puede que modifique algo del Readme pero la RC1 es funcional y contiene respuesta a todos los apartados.
